src/components/Footer/index.tsx

```diff
@@ -18,7 +18,7 @@ const Footer: React.FC = () => {
           <Typography variant="body/large">About Us</Typography>
         </Link>
         <Link href="https://acmucsd.com/">
-          <Image src="/acm-logo.png" alt="ACM" width={144} height={144} />
+          <Image src="/assets/acm-logo.png" alt="ACM" width={144} height={144} />
```

Comment:
might it be better to import the png file directly? like what is done for `SlidingDiamond`? I think it exports an object with metadata that Next.JS is able to use for certain optimizations